Automattic/wp-calypso
# FSE/Gutenberg: Add link color to block editor

#### Changes proposed in this Pull Request

* Will address #41844 (requires deploy)
* Closes #41906 (superseded)

Add anchor color `#0073aa` to the block editor styles. This ensures that themes that do not define a link color in their editor styles will not show links in the same color as text.

Prototyped in D43160-code

#### Testing instructions

1. Apply patch ()
1. Sandbox `example.wordpress.com`
1. Apply theme "Independent Publisher 2", "Penscratch 2", or "TwentyTwenty"
1. Load the editor `/wp-admin/post-new.php`
1. Add a link in a paragraph, text should be blue (not black like in the #41844).
1. Try activating a different plugin version (`7.8.1` or `8.1.0-rc.1`). The style should not be added. `8.1.0-rc.1` does not include the fix, but Gutenberg `master` does.

#### Screens

With TwentyTwenty theme

##### Before

![before](https://user-images.githubusercontent.com/841763/81419200-2c9a2e80-914e-11ea-9167-8c31da98e07d.png)

##### After
![after](https://user-images.githubusercontent.com/841763/81419211-302db580-914e-11ea-9834-fe4e03423bcf.png)
